curvefs/src/mds/space/volume_space.cpp

```diff
@@ -267,6 +286,22 @@ SpaceErrCode VolumeSpace::AcquireBlockGroup(uint64_t blockGroupOffset,
 SpaceErrCode VolumeSpace::AcquireBlockGroupInternal(uint64_t blockGroupOffset,
                                                     const std::string& owner,
                                                     BlockGroup* group) {
+    if (owner.empty()) {
```

Comment:
Maybe it can be more readable:

```C++
    if (owner.empty()) {
        auto it = deallocatingGroups_.find(blockGroupOffset);
        if (it == deallocatingGroups_.end()) {
            return SpaceErrNotFound;
        }
        
        *group = it->second;
        VLOG(6) << "VolumeSpace recieve acquire blockgroup="
                << group->DebugString()
                << " rquest from metaserver, current block group is under "
                    "deallocating";
        return SpaceOk;
    }
```

Reply:
good idea. fix.